### ISSUE TITLE
One small bug?

### ISSUE THREAD

**Issue by DrugD** [PaddlePaddle/Paddle#34297]
部分日志：
[07/21 14:40:44] ppdet.engine INFO: Epoch: [0] [  0/200] learning_rate: 0.000000 loss_xy: 6.467412 loss_wh: 22.083004 loss_iou: 25.574692 loss_obj: 5243.871582 loss_cls: 33.527027 loss: 5331.523438 eta: 27 days, 22:18:02 batch_cost: 18.5622 data_cost: 17.9648 ips: 1.7239 images/s
[07/21 14:43:00] ppdet.engine INFO: Epoch: [0] [ 20/200] learning_rate: 0.000025 loss_xy: 7.020596 loss_wh: 23.817680 loss_iou: 27.600710 loss_obj: 947.480469 loss_cls: 35.948715 loss: 1037.630493 eta: 10 days, 19:39:33 batch_cost: 6.6231 data_cost: 6.0753 ips: 4.8315 images/s
[07/21 14:45:21] ppdet.engine INFO: Epoch: [0] [ 40/200] learning_rate: 0.000050 loss_xy: 6.664555 loss_wh: 18.636883 loss_iou: 26.876186 loss_obj: 72.916740 loss_cls: 28.785061 loss: 155.548477 eta: 10 days, 14:16:13 batch_cost: 6.8879 data_cost: 6.3277 ips: 4.6458 images/s
[07/21 14:47:48] ppdet.engine INFO: Epoch: [0] [ 60/200] learning_rate: 0.000075 loss_xy: 5.925452 loss_wh: 14.537316 loss_iou: 25.043835 loss_obj: 43.707775 loss_cls: 23.676754 loss: 113.562012 eta: 10 days, 15:33:38 batch_cost: 7.1558 data_cost: 6.5692 ips: 4.4719 images/s
[07/21 14:50:08] ppdet.engine INFO: Epoch: [0] [ 80/200] learning_rate: 0.000100 loss_xy: 6.217870 loss_wh: 14.154335 loss_iou: 26.165279 loss_obj: 41.176506 loss_cls: 20.592834 loss: 109.111626 eta: 10 days, 13:37:16 batch_cost: 6.8671 data_cost: 6.3141 ips: 4.6599 images/s
[07/21 14:52:32] ppdet.engine INFO: Epoch: [0] [100/200] learning_rate: 0.000125 loss_xy: 5.697371 loss_wh: 12.644457 loss_iou: 23.891020 loss_obj: 32.444725 loss_cls: 15.826570 loss: 90.156052 eta: 10 days, 13:11:58 batch_cost: 6.9742 data_cost: 6.4142 ips: 4.5884 images/s

-----------

eta的时间算错了？  还是？？我的epoch设为10个。eta应该是3～4h就可以。

**Comment by Superjomn** [comment]
你好，请问这个是 paddle detection 吗？

麻烦帮提供下， paddlepaddle 和对应组件的 版本信息可以吗？
